search_params.py

```diff
@@ -141,7 +141,10 @@ def init_search_algorithm(search_alg, metric=None, mode=None):
         from ray.tune.search.bayesopt import BayesOptSearch
 
         return BayesOptSearch(metric=metric, mode=mode)
-    logging.info(f"{search_alg} search is found, run BasicVariantGenerator().")
+    elif search_alg == "basic_variant":
+        pass
+    else:
+        logging.info(f"No Search algorithm is found. Will run basic_variant")
```

Comment:
Why add these codes?

Reply:
The former logic could print weird stuff if user inputs something like "abcdefg"

Reply:
I forgot to change this part... My original thought was to inform users of the search algorithm they use...